docs/reports/2024-05-07-v1-upgrade.md

```diff
@@ -0,0 +1,56 @@
+
+# Puffer Protocol Upgrade (v1.0)
+
+Puffer Protocol Upgrade (v1.0) is a major upgrade that introduces the protocol and new version of the vault (pufETH v2). This post will cover the details of the upgrade and the changes that are made.
+
+
+# Upgrade Overview
+This upgrade will introduce the new version of the vault (pufETH v2) and the new version of the PufferPool (v1). The new vault will support deposits for ETH, stETH, wstETH, and wETH, as well as, integrations with EigenLayer and Lido withdrawal queue. For more information on what is included in this update please check out the latest docs: 
```

Comment:
wStETH deposit is done via proprietary `PufferDepositor`, it unwraps it to stETH and deposits as stETH to the Vault.

```suggestion
This upgrade will introduce the new version of the vault (pufETH v2) and the new version of the PufferPool (v1). The new vault will support deposits for ETH, stETH, and WETH, as well as, integrations with EigenLayer and Lido withdrawal queue. For more information on what is included in this update please check out the latest docs: 
```